cwltool/process.py

```diff
@@ -279,9 +279,11 @@ def moveIt(src, dst):
         if src != dst:
             _logger.debug("Copying %s to %s", src, dst)
             if os.path.isdir(src):
+                if os.path.exists(dst):
+                    shutil.rmtree(dst)
```

Comment:
Could you add a check if `dst` is a directory.  If it isn't a directory, it should use os.unlink() instead.  (If you try to use `shutil.rmtree` on a regular file it will raise an exception).

Reply:
Thanks @tetron. The isdir function is prob more appropriate in this case anyway. Not sure I understand the scenario where a src dir could equate to a dst file. The copy2 func will allow overwrites so no need to unlink in the else block.

Reply:
If `src` is a directory but `dst` is a file, `copytree()` will fail.  I believe the problem you're trying to solve is correctly overwriting files in the final directory.

Reply:
That's fair. Committed for completeness. ;)